src/scss/components/buttons.scss

```diff
@@ -1,5 +1,5 @@
 // Add all variations for buttons
-.sidebar-item.formbutton-edit {
+.formbutton-edit {
```

Comment:
Have in mind that this button `formbutton-edit` might exist in different places and removing this specificity might break the styles.